code/game/machinery/rechargestation.dm

```diff
@@ -156,9 +157,10 @@
 							O.reagents.add_reagent("enzyme", 2 * coeff)
 					//Medical
 					if(istype(O,/obj/item/weapon/reagent_containers/glass/bottle/robot))
-						var/obj/item/weapon/reagent_containers/glass/bottle/robot/B = O
-						if(B.reagent && (B.reagents.get_reagent_amount(B.reagent) < B.volume))
-							B.reagents.add_reagent(B.reagent, 2 * coeff)
+						var/obj/item/weapon/reagent_containers/glass/bottle/robot/B = O //B = 0 - Changed the 'B's to '0's below
+						if(O.reagent && (O.reagents.get_reagent_amount(O.reagent) < O.volume))
+							O.reagents.add_reagent(O.reagent, 2 * coeff)
+							O:update_icon()

```

Comment:
eerrr.. why would you use O? It was already neatly typepathed as a beaker, without a nasty colon operator, just call B.update_icon()
and this is also causing travis to fail because O.reagent is not a var